roles/wazuh/ansible-wazuh-manager/tasks/main.yml

```diff
@@ -7,6 +7,34 @@
       - tar
     state: present
 
+- name: Check if NodeJS service exists
+  stat:
+    path: /usr/bin/node
+  register: node_service_status
+
+- name: Install NodeJS repository
+  block:
+    - name: Download NodeJS repository script
+      get_url:
+        url: "https://{{ repo_dic[ansible_os_family|lower] }}.nodesource.com/setup_8.x"
```

Comment:
Node is maintaining the syntax since version 4, moving `nodesource.com/setup_8.x` to a variable under the `nodejs` variable stated in the previous comment would ease the process of upgrading when we move to node 9

Reply:
Hi @jm404 !

Find the changes in https://github.com/wazuh/wazuh-ansible/pull/330/commits/e003a2d51eb3c1fad46baa5e7644ad727796898d

Kr,

Rshad